tests/sentry/monitors/endpoints/test_monitor_ingest_checkin_index.py

```diff
@@ -182,6 +182,44 @@ def test_monitor_creation_via_checkin(self):
             checkins = MonitorCheckIn.objects.filter(monitor=monitor)
             assert len(checkins) == 1
 
+    def test_monitor_environment_creation_via_checkin(self):
```

Comment:
I think this test doesn't actually need to test environments, I think we jsut needed a test that creates two checkins in a row, and ensure the first creates and the second updates.

Reply:
Ah you're right it still fails. Will update + rename